Neos.Media.Browser/Classes/Controller/AssetController.php

```diff
@@ -623,7 +623,7 @@ public function deleteAction(Asset $asset): void
         }
 
         $this->assetRepository->remove($asset);
-        $this->addFlashMessage('assetHasBeenDeleted', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
+        $this->addFlashMessage('assetHasBeenDeleted', '', Message::SEVERITY_OK, [str_replace($asset->getLabel(), '%', '%%')], 1412375050);
```

Comment:
`htmlspecialchars` is missing here - we should add it too

Reply:
Thx! Added.